ts/sagas/notifications.ts

```diff
@@ -54,25 +44,21 @@ function* updateInstallation(): Iterator<Effect> {
       // Send the request to the backend
       const backendClient = BackendClient(apiUrlPrefix, sessionToken);
 
-      try {
-        const response:
-          | BasicResponseType<Installation>
-          | undefined = yield call(backendClient.createOrUpdateInstallation, {
-          id: notificationsInstallation.uuid,
-          installation: {
-            platform: notificationsPlatform,
-            pushChannel: notificationsInstallation.token
-          }
-        });
-
-        /**
-         * If the response is undefined (can't be decoded) or the status is not 200 notify the problem
-         */
-        if (!response || response.status !== 200) {
-          notifyNotificationsInstallationError();
+      const response:
+        | BasicResponseType<NonEmptyString>
+        | undefined = yield call(backendClient.createOrUpdateInstallation, {
```

Comment:
please add story to have this call retry e couple of times in case of temporary errors (assign it to me as I will add that functionality to the client), thanks